shell/list/node.vue

```diff
@@ -164,22 +188,58 @@ export default {
       <template #sub-row="{fullColspan, row, onRowMouseEnter, onRowMouseLeave}">
         <tr
           class="taints sub-row"
-          :class="{'empty-taints': !row.spec.taints || !row.spec.taints.length}"
+          :class="{'empty-taints': !displayTaintsAndLabels(row)}"
           @mouseenter="onRowMouseEnter"
           @mouseleave="onRowMouseLeave"
         >
-          <template v-if="row.spec.taints && row.spec.taints.length">
+          <template v-if="displayTaintsAndLabels(row)">
             <td>&nbsp;</td>
             <td>&nbsp;</td>
             <td :colspan="fullColspan-2">
-              {{ t('node.list.nodeTaint') }}:
-              <Tag
-                v-for="taint in row.spec.taints"
-                :key="taint.key + taint.value + taint.effect"
-                class="mr-5"
-              >
-                {{ taint.key }}={{ taint.value }}:{{ taint.effect }}
-              </Tag>
+              <span v-if="row.spec.taints && row.spec.taints.length">
+                {{ t('node.list.nodeTaint') }}:
+                <Tag
+                  v-for="taint in row.spec.taints"
+                  :key="taint.key + taint.value + taint.effect"
+                  class="mr-5 mt-2"
+                >
+                  {{ taint.key }}={{ taint.value }}:{{ taint.effect }}
+                </Tag>
+              </span>
+              <span
+                v-if="!!displayLabels(row).length"
+                class="mt-5"
+              > {{ t('node.list.nodeLabels') }}:
+                <span
+                  class="mt-5 labels"
+                >
+                  <span v-if="!isLabelsVisible">
+                    <Tag
+                      v-for="(label, i) in displayLabels(row).slice(0, 7)"
```

Comment:
This operation of `slice` is a pattern we should avoid directly on the templates. With the computed, this should be included there.